lib/assets.js

```diff
@@ -12,14 +12,14 @@ var defaults = {
   }
 };
 
-module.exports = function (gulp, options) {
-  var opts = merge({}, defaults, options);
-  var dest = opts.dest;
-  var optimize = opts.optimize;
-  var plugins = opts.plugins;
-  var src = opts.src;
+module.exports = (gulp, options) => {
+  const opts = merge({}, defaults, options);
```

Comment:
Thanks for the feedback @erikjung! Could there be a case made for using `let` instead? I was under the impression that my options were either `let` or `const` for ES2015/ES6. I didn't realize I could still use `var`. I'm going to search on the Googlez so I can better understand this.